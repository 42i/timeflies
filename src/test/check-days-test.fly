wp a

day 2012-09-12 10 15
- a 2

day 2012-09-13 11 14
- a 3

day 2012-09-14 12 12:30
- a 1:15