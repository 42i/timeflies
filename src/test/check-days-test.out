Day check (all):
2012-09-12 Wed: worked  5.00, allocated  2.00, delta -3.00
2012-09-14 Fri: worked  0.50, allocated  1.25, delta  0.75
2 problems detected.
